frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -67,12 +67,15 @@ const Metric = (props: MetricProps) => {
     const theme = useTheme();
 
     const data = useMemo(() => {
+        const mode = (props.type === "none") ? [] : ["gauge"];
+        showValue && mode.push("number");
+        (delta !== undefined) && mode.push("delta");
```

Comment:
(format document) would remove the parenthesis

Reply:
it does in my VS ...

Reply:
Show me the settings.json one of these days.
Honestly, I prefer to keep the parenthesis that improve the reading at no runtime cost.